lib/redmine/menu_manager/menu_helper.rb

```diff
@@ -157,14 +155,14 @@ def render_menu_node(node, project=nil)
     end
   end
 
-  def render_menu_node_with_children(node, project=nil)
+  def render_menu_node_with_children(node, project = nil)
```

Comment:
Assignment Branch Condition size for render_menu_node_with_children is too high. [18.49/15]